CHANGELOG.rst

```diff
@@ -1,3 +1,8 @@
+# 2.12.0
+
+* Refactor: Rename LibraryCollectionKey to LibraryElementKey.
```

Comment:
Nit: we have been using the term "item" in a few place to refer to any thing in a library (block/collection/container) so I would slightly prefer that over "element"

Reply:
Updated here https://github.com/openedx/opaque-keys/pull/369/commits/fc0cff1bc932993f47f6fc0b58bb3cd882a1ca31